datanode/gateway/graphql/fees_stats_resolver.go

```diff
@@ -43,6 +43,10 @@ func (r *feesStatsResolver) Epoch(_ context.Context, obj *eventspb.FeesStats) (i
 	return int(obj.EpochSeq), nil
 }
 
+func (r *feesStatsResolver) TotalMakeFeesReceived(_ context.Context, obj *eventspb.FeesStats) ([]*eventspb.PartyAmount, error) {
+	return obj.TotalMakerFeesReceived, nil
+}
```

Comment:
Why did you have to add this method while none of the others are needed but have the same type?